.github/workflows/nerdctl.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
     env:
       JOB_NAME: "nerdctl-${{ matrix.deployment }}-${{ matrix.ipFamily }}"
       IP_FAMILY: ${{ matrix.ipFamily }}
-      NERDCTL_VERSION: "1.7.4"
+      NERDCTL_VERSION: "2.0.2"
```

Comment:
do you think we need to be testing both? were there major breaking changes in 2.x vs 1.x?

Reply:
As for the functions used in kind, there are no major changes except the support of containerd v2.

I'm fine if you want to test both of them.

Reply:
sounds like we don't need to for now, thanks